tests/subset.rs

```diff
@@ -54,13 +54,18 @@ fn test_subset<A: Adversary<Subset<NodeId>>>(
     }
     let output = expected.unwrap();
     assert!(once(&output).eq(network.observer.outputs()));
-    // The Subset algorithm guarantees that more than two thirds of the proposed elements
-    // are in the set.
-    assert!(output.len() * 3 > inputs.len() * 2);
-    // Verify that the set's elements match the proposed values.
-    for (id, value) in output {
-        assert_eq!(inputs[&id], value);
-    }
+    match output {
```

Comment:
I think that in the test each node should output 5 same contributions - one for each node - and then `Done`. On master every node outputs a map:
```
[{NodeId(0): [65, 108, 112, 104, 97], NodeId(1): [66, 114, 97, 118, 111], NodeId(2): [67, 104, 97, 114, 108, 105, 101], NodeId(3): [68, 101, 108, 116, 97], NodeId(4): [69, 99, 104, 111]}]
```
In your test I see all outputs having only one contribution and then `Done`. Other contributions are missing.